packages/cyberstorm/src/components/Tooltip.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { PropsWithChildren } from "react";
+import * as RadixTooltip from "@radix-ui/react-tooltip";
+import styles from "./componentStyles/Tooltip.module.css";
+
+export interface TooltipProps {
+  label?: string;
```

Comment:
Could this be `ReactNode`? I don't think we'll only restrict tooltips to simple strings.